lisa/operating_system.py

```diff
@@ -521,7 +525,7 @@ def _get_os_version(self) -> OsVersion:
             no_error_log=True,
         )
         if cmd_result.exit_code == 0 and cmd_result.stdout != "":
-            for vendor in ["Fedora", "CentOS", "Red Hat", "XenServer"]:
+            for vendor in ["Fedora", "CentOS", "Red Hat", "XenServer", "AlmaLinux"]:
```

Comment:
Over Check if the list of string is needed. This method is overridden in Redhat already, it just need to check whether it's fedora. If so, fix it in another commit.

Reply:
Yes, this method is overridden in Redhat already.

Reply:
Yes, please fix it. Remove duplicate code in a separated commit.

Reply:
Done